pkg/core/experiment_run.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+import (
+	"context"
+	"time"
+)
+
+// ExperimentRunStore defines operations for working with experiment runs
+type ExperimentRunStore interface {
```

Comment:
how about rename the interface? for example `ExperimentStatusStore`?

Reply:
Wouldn't status be misleading since it sounds singular but there would be many such "status" for a single experiment